get-config.js

```diff
@@ -0,0 +1,20 @@
+function readConfig() {
+	var config = {};
+
+	try {
+		config = require( './config' );
+	} catch ( e ) {
+		console.warn( 'no configuration file present' );
+	}
+
+	return config;
+};
+
+function getConfig() {
```

Comment:
what if we passed in the file path as an argument instead of hard-coding `./config`? would we have a good reason to develop or test with multiple config files?